### ISSUE TITLE
Help with installation

### ISSUE THREAD

**Issue by buthig666** [Nonary/ResolutionAutomation#30]
I would love a little guidance.  I have followed the directions to a T about a dozen times and can't for the life of me get the resolution to change at all.  Host is Win 11 4k @120.  My clients are 4k @60- android tvs, 1080p @60- laptop and a phone that is 2520 x 1080 @90.  Everything stays the same as my host no matter what.  On vrs 2.2 and Sunshine 0.21.

There is no request for elevated rights when installing the .bat.  

I may be mistaken but there used to be a mention of Sunshine's log file being set to debug.  I don't see that in the install instructions anymore?  Any help would be appreciated.

I realize this probably isn't the best place to ask for help, is there a Discord or Reddit channel I could go to?

thanks

**Comment by Nonary** [comment]
> I may be mistaken but there used to be a mention of Sunshine's log file being set to debug. I don't see that in the install instructions anymore? Any help would be appreciated.

That is no longer required if you have the latest Sunshine installed (0.21.0)

if you are using an older version of sunshine you would need to download older releases

**Comment by Nonary** [comment]
Upload the log file of the resolution script here

**Comment by buthig666** [comment]
Nonary, thank you for responding.  I have been in and out of Hospital this week and over the weekend I should be home and will sit down for a few hours and try again, and upload logs.  For now I have uninstalled the bat because I am using my Laptop from hospital to access Host at home and don't want to break anything.  Thanks again.

**Comment by buthig666** [comment]
Ok, I'm home.  I'm sorry to be so dunce but I don't know where the script log file is.  The only text file in the staging folder is the overrides document.

**Comment by buthig666** [comment]
cannot comment at this time ???

**Comment by buthig666** [comment]
[sunshine log.txt](https://github.com/Nonary/ResolutionAutomation/files/13042049/sunshine.log.txt)
 
Here it is in a txt file.  So I ran uninstall bat.  Deleted Res. Auto folder.  Updated Host and Clients Sun and Moon respectively.  Downloaded the latest version of Res. Auto and ran install bat.  No special permissions were requested.  I restarted host and connected with 3 different clients with 3 different resolutions.  All remained the same as host.  I used to use a dummy Hdmi dongle but no longer use it as my monitor allows me to stream with Hdmi 2.1 when it is turned off.  My fear is that by using CRU months ago there are left over ghosts in the shell.  Thank you so much for your patience, your help and sharing your work.

**Comment by Nonary** [comment]
> [sunshine log.txt](https://github.com/Nonary/ResolutionAutomation/files/13042049/sunshine.log.txt)
> 
> Here it is in a txt file. So I ran uninstall bat. Deleted Res. Auto folder. Updated Host and Clients Sun and Moon respectively. Downloaded the latest version of Res. Auto and ran install bat. No special permissions were requested. I restarted host and connected with 3 different clients with 3 different resolutions. All remained the same as host. I used to use a dummy Hdmi dongle but no longer use it as my monitor allows me to stream with Hdmi 2.1 when it is turned off. My fear is that by using CRU months ago there are left over ghosts in the shell. Thank you so much for your patience, your help and sharing your work.

I don't see you trying to stream on this log, keep in mind if you restart Sunshine it purges the log. Please try streaming again, then upload the file from C:\Windows\Temp\sunshine.log

Thanks

**Comment by billyjacoby** [comment]
I'm experiencing the same issue as stated here. I've run the install file without issue, but no matter what when I try to launch the Desktop (or any other) app via moonlight, the resolution is still for my connected ultrawide (3440x 1440) instead of what Moonlight is sending. 

Here are the logs:
[log.txt](https://github.com/Nonary/ResolutionAutomation/files/13196723/log.txt)

**Comment by buthig666** [comment]
@billyjacoby Where did you add custom resolutions?  Nvidia control panel?  My resolutions are already there except for phone which I have added manually.

**Comment by billyjacoby** [comment]
@buthig666 Yeah exactly, I added them first and didn't have the boxes checked which took me way longer to figure out than I'd like to admit 😅

**Comment by Scrutape** [comment]
Also having problems getting this working, despite following the instructions to the letter. When I check the log, it shows the received resolution was my monitor's resolution, despite me using Moonlight on a Steamdeck.

**Comment by antonmorch** [comment]
> Also having problems getting this working, despite following the instructions to the letter. When I check the log, it shows the received resolution was my monitor's resolution, despite me using Moonlight on a Steamdeck.

@Scrutape. I had this problem with the deck as well. I deleted the text in the override file, and then it started working. Hope it helps.

**Comment by SebbeJohansson** [comment]
Im having the same issue and reming it like @antonmorch said helps. But I do want to keep the resolution higher than what the steamdeck has. For example, cyberpunk doesn't work at 1280x800. Ive tried setting it both to the one that is in the example, and also 1920x1200, but no dice.

@Nonary is there something wrong with the override functionality?

**Comment by SebbeJohansson** [comment]
Heh. I guess I just needed to comment to figure it out myself.

What I needed to do was to remove the "comment" in the txt file. I assume that isn't supported in whatever method is used to read the .txt file.
![image](https://github.com/Nonary/ResolutionAutomation/assets/5083273/7b9020d8-1986-47c9-9789-46235e6fe944)

Also, another note if someone finds this thread, to get the script to work at all I had to go into properties for the folder and make it, and everything in it, non-read-only.
![image](https://github.com/Nonary/ResolutionAutomation/assets/5083273/963b8483-3182-4317-8d34-24de26bebf96)

**Comment by wigglyturtle** [comment]
[log.txt](https://github.com/Nonary/ResolutionAutomation/files/13828049/log.txt)

Hello, I'm having a similiar issue. It looks like it executes the script just fine but the resolution never actually changes.
